packages/codemods/package.json

```diff
@@ -33,8 +33,8 @@
     "test:ui": "vitest --ui --coverage --typecheck"
   },
   "dependencies": {
-    "@commander-js/extra-typings": "^12.1.0",
-    "commander": "^12.1.0",
+    "@commander-js/extra-typings": "^13.0.0",
+    "commander": "^13.0.0",
```

Comment:
I mentioned @gwansikk because of [this comment](https://github.com/toss/suspensive/pull/1407#issuecomment-2597829042)
Could you double check that this change is okay?